tests/tools/tflite_comparator/src/tflite_comparator.cc

```diff
@@ -342,30 +371,19 @@ int main(const int argc, char **argv)
         break;
       case NNFW_TYPE_TENSOR_UINT8:
       case NNFW_TYPE_TENSOR_QUANT8_ASYMM:
-        matched = compareBuffersExact<uint8_t>(ref_output.uint8, output, out_idx);
+        matched = compareBuffers<uint8_t>(ref_output.uint8, output, out_idx);
         break;
       case NNFW_TYPE_TENSOR_QUANT8_ASYMM_SIGNED:
-        matched = compareBuffersExact<int8_t>(ref_output.int8, output, out_idx);
+        matched = compareBuffers<int8_t>(ref_output.int8, output, out_idx);
         break;
       case NNFW_TYPE_TENSOR_INT32:
-        matched = compareBuffersExact<int32_t>(ref_output.i32, output, out_idx);
+        matched = compareBuffers<int32_t>(ref_output.i32, output, out_idx);
         break;
       case NNFW_TYPE_TENSOR_FLOAT32:
-        // TODO better way for handling FP error?
-        for (uint32_t e = 0; e < num_elems(&ti); e++)
-        {
-          float refval = ref_output.f[e];
-          float val = reinterpret_cast<const float *>(output.data())[e];
-          if (std::abs(refval - val) > max_float_difference)
-            max_float_difference = std::abs(refval - val);
-
-          matched = nnfw::misc::fp32::absolute_epsilon_equal(refval, val)
-                      ? true
-                      : nnfw::misc::fp32::epsilon_equal(refval, val, tolerance);
-        }
```

Comment:
Here was a bug that it return true if the last element is same and other elements are different.